data/presets/presets/highway/motorway.json

```diff
@@ -5,6 +5,7 @@
         "ref_road_number",
         "oneway_yes",
         "maxspeed",
+        "maxspeed_advisory",
```

Comment:
Normally advisory speed limits appear on `motorway_link` and other `*_link` roads. It can happen on non-`link`s too, but typically only at curves.

Reply:
Ok so i have to add this field to all *link roads or something else that i should do.

Reply:
Yes, it would make sense to add this field to all the `*_link` presets. Maybe even remove `maxspeed` in favor of `maxspeed_advisory` on those presets.

The non-`link` presets should be consistent with each other. So if `motorway` has it, then the other `highway` presets like `residential` and `tertiary` should too. For these presets, an extra “Advisory Speed Limit” field could be confusing if it appears by default, since most roads don’t have advisory speed limits. I wonder if we should leave it off of these non-`link` presets until #4871 is implemented.